conanfile.py

```diff
@@ -10,11 +10,11 @@ class NaiveTsearchConan(ConanFile):
     homepage = "https://github.com/kulp/naive-tsearch"
     url = "https://github.com/kulp/naive-tsearch"
     license = "MIT"
-    exports_sources = "CMakeLists.txt", "README.md", "*.c", "*.h", "*.in", "*.inc", "LICENSE"
+    exports_sources = "CMakeLists.txt", "README.md", "*.c", "*.h", "*.in", "*.inc", "LICENSE", "tests/**"
     exports = "LICENSE"
     no_copy_source = True
     settings = "os", "arch", "compiler", "build_type"
-    requires = "boost/1.72.0"
+    requires = "boost/1.73.0"
```

Comment:
Is the requirement on boost actually increased ?

Reply:
No, this is just increasing the boost version used for building with conan.
conan has a syntax for version ranges though: `boost/[>1.70.0]`.
Maybe use that instead of having to bump this version too often.

Reply:
May we can use the version range in the future. Thanks for explaining.

Reply:
Yup, I'll do it in a next pr.